recipes/imas-data-dictionary/recipe.yaml

```
@@ -0,0 +1,62 @@
+context:
+  version: "4.0.1"
+  name: "imas-data-dictionary"
+  build: 0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name | split("-") | join("_") }}-${{ version }}.tar.gz
+  sha256: 89327fabc71cd06772833c04dab06800ed3ac41ec7b9a671227881d429690fe7
+
+build:
+  number: ${{ build }}
+  noarch: python
+  python:
+    entry_points:
+      - idsinfo = data_dictionary.idsinfo:main
+  files:
+    include: ["**"]
+    exclude:
+      - dd_${{ version }}/share/
+
+requirements:
+  build:
+    - saxon-he
+    - libxslt
```

```suggestion
    - libxslt
    - sed
```
Just add `sed` to the build dependencies here so that you don't have to use a different command for macOS than Linux.